Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
```

Comment:
### [pg](https://github.com/ged/ruby-pg)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.3.5 → 1.4.3

[_[change-log](https://github.com/ged/ruby-pg/blob/master/History.rdoc), [source-code](https://github.com/ged/ruby-pg), [gem-diff](https://my.diffend.io/gems/pg/1.3.5/1.4.3)_]





<details>
<summary>Commits</summary>

A change of **68** commits. See the full changes on [the compare page](https://github.com/ged/ruby-pg/compare/v1.3.5...v1.4.3).

These are the first 10 commits:
- (cc503ea) [Handle client error after all data consumed in #copy_data fo…](https://github.com/ged/ruby-pg/commit/cc503eae9dc83440bb679af143b5a29beedc0a1b)
- (f681523) [Fix for TruffleRuby, raise original exception explicitly](https://github.com/ged/ruby-pg/commit/f6815239dddb610d01587d7b6c3198ebf96ba191)
- (a40dd7b) [Merge pull request #455 from stevewoodcock/ignore-get-copy-d…](https://github.com/ged/ruby-pg/commit/a40dd7ba9860119d9c8a988d829c7d1a0f71d9f8)
- (ee77ed4) [Add block-call semantics to PG::Connection.new](https://github.com/ged/ruby-pg/commit/ee77ed4320160bb1d14f6f161dd8a3af2e65ec59)
- (0242f31) [Avoid spurious keyword argument warning on Ruby 2.7](https://github.com/ged/ruby-pg/commit/0242f3194c15d10ad5037d4581e78cb70b13b648)
- (2187b13) [Merge pull request #456 from jeremyevans/avoid-ruby27-keywor…](https://github.com/ged/ruby-pg/commit/2187b1366f357cb154a70df07aecfcd9e024b90c)
- (aeaf535) [Add a useful example to conn.status](https://github.com/ged/ruby-pg/commit/aeaf535713553763b99fa4546518758fcd68e001)
- (0ab7533) [Add PG::Connection#hostaddr](https://github.com/ged/ruby-pg/commit/0ab753366056cd53279df48bdc85e021f4cf1797)
- (bdd1656) [Improve API documentation to conn.host](https://github.com/ged/ruby-pg/commit/bdd16568d0e1edf1bbb765f54cfc07c210e2565f)
- (f14c94a) [Add conn.conninfo_parse to wrap PQconninfoParse](https://github.com/ged/ruby-pg/commit/f14c94aa2ddc25b74310ecc7420a135eababb1e8)


</details>